src/functions-templates/javascript/google-analytics/package.json

```diff
@@ -19,6 +19,7 @@
   "author": "Netlify",
   "license": "MIT",
   "dependencies": {
+    "netlify-cli": "latest",
```

Comment:
What is the reason we need the CLI as a dependency for this function template?
Function templates are used for this command:
https://cli.netlify.com/commands/functions#functionscreate

to create a new function from a template.